src/renderer/component/fileCard/view.jsx

```diff
@@ -1,110 +1,100 @@
-    } else if (claim === null) {
-      description = __('This address contains no content.');
-    }
+    // Come back to this on other pages
+    // let description = '';
+    // if (isResolvingUri && !claim) {
+    //   description = __('Loading...');
+    // } else if (metadata && metadata.description) {
+    //   description = metadata.description;
+    // } else if (claim === null) {
+    //   description = __('This address contains no content.');
+    // }
 
+    // We don't want to allow a click handler unless it's in focus
+    // I'll come back to this when I work on site-wide keyboard navigation
+    /* eslint-disable jsx-a11y/click-events-have-key-events */
     return (
       <section
-        className={`card card--small card--link ${obscureNsfw ? 'card--obscured ' : ''}`}
-        onMouseEnter={this.handleMouseOver.bind(this)}
-        onMouseLeave={this.handleMouseOut.bind(this)}
+        tabIndex="0"
+        role="button"
+        onClick={() => navigate('/show', { uri })}
+        className={classnames('card card--small card__link', {
+          'card--obscured': shouldObscureNsfw,
+        })}
       >
-        <div className="card__inner">
-          <Link onClick={() => navigate('/show', { uri })} className="card__link">
-            <CardMedia title={title} thumbnail={thumbnail} />
-            <div className="card__title-identity">
-              <div className="card__title" title={title}>
-                <TruncatedText lines={1}>{title}</TruncatedText>
-              </div>
-              <div className="card__subtitle">
-                <span className="card__indicators card--file-subtitle">
-                  <FilePrice uri={uri} /> {isRewardContent && <Icon icon={icons.FEATURED} />}{' '}
-                  {fileInfo && <Icon icon={icons.LOCAL} />}
-                </span>
-                <span className="card--file-subtitle">
-                  <UriIndicator uri={uri} link span smallCard />
-                </span>
-              </div>
-            </div>
-          </Link>
-          {/* Test for nizuka's design: should we remove description?
-            <div className="card__content card__subtext card__subtext--two-lines">
-              <TruncatedMarkdown lines={2}>{description}</TruncatedMarkdown>
+        <CardMedia thumbnail={thumbnail} />
+
+        <div className="card__title-identity">
+          <div className="card__title--small">
+            <TruncatedText lines={3}>{title}</TruncatedText>
+          </div>
+
+          <div className="card__subtitle">
+            <UriIndicator uri={uri} link />
+            <div className="card--file-subtitle">
+              <FilePrice uri={uri} /> {isRewardContent && <Icon icon={icons.FEATURED} padded />}
+              {fileInfo && <Icon icon={icons.LOCAL} padded />}
             </div>
-            */}
+          </div>
         </div>
-        {obscureNsfw && this.state.hovered && <NsfwOverlay />}
+        {obscureNsfw && <NsfwOverlay />}
```

Comment:
still need to figure this out. will get it when we determine proper colors/how this will look. ignore for now